flake.nix

```diff
@@ -470,31 +557,31 @@
             arm64 = wasmcloud-aarch64-unknown-linux-musl-oci-wolfi;
           };
         in
-          packages
+          (concatMapAttrs (_: {packages, ...}: packages) attrs)
           // {
             inherit
-              wash-oci-debian
-              wash-oci-wolfi
-              wasmcloud-oci-debian
-              wasmcloud-oci-wolfi
+              wash-aarch64-unknown-linux-gnu-fhs
               wash-aarch64-unknown-linux-musl-oci-debian
               wash-aarch64-unknown-linux-musl-oci-wolfi
+              wash-oci-debian
+              wash-oci-wolfi
+              wash-riscv64gc-unknown-linux-gnu-fhs
+              wash-x86_64-unknown-linux-gnu-fhs
               wash-x86_64-unknown-linux-musl-oci-debian
               wash-x86_64-unknown-linux-musl-oci-wolfi
               wasmcloud-aarch64-unknown-linux-gnu-fhs
               wasmcloud-aarch64-unknown-linux-musl-oci-debian
               wasmcloud-aarch64-unknown-linux-musl-oci-wolfi
+              wasmcloud-oci-debian
+              wasmcloud-oci-wolfi
               wasmcloud-riscv64gc-unknown-linux-gnu-fhs
               wasmcloud-x86_64-unknown-linux-gnu-fhs
               wasmcloud-x86_64-unknown-linux-musl-oci-debian
               wasmcloud-x86_64-unknown-linux-musl-oci-wolfi
               ;
 
+            default = attrs.wasmcloud.packages.wasmcloud;
             rust = hostRustToolchain;
-            wash = pkgs.runCommandLocal "wash" {} ''
```

Comment:
Can we not keep around a `wash` pkg alias still for using this in downstream flakes? Looks like we already have it available in `attrs` now anyway. Not a super big thing, but I had found it helpful when pulling into other flakes

Reply:
We have a `wash` output, which only builds `wash` now